### PR TITLE
ULTRA decrease large amounts of vehicles from traffic distributions

### PR DESCRIPTION
**Pull request by JenishPatel99**
There were instances inside `distributions.py` where the number of vehicles for a single route was set to 1000. However, during a normal run with out baselines, we have never exhausted all 1000 of vehicles. I do feel it is unnecessary to have these many. This will also help decrease the scenario generation as we have less vehicles to construct and ID. Let me know your thoughts on this @christianjans

### CODE REVIEW

File: ultra/ultra/scenarios/common/distributions.py [hunk 1]
```diff
@@ -146,7 +146,7 @@
             },
             "south-north": None,  # blocking
             "west-east": {
-                "vehicles": 1000,
+                "vehicles": 200,
```
Comment (christianjans) [inline ultra/ultra/scenarios/common/distributions.py:149]: Interesting, okay good call. How did you decide on the number 200?

Comment (JenishPatel99) [reply ultra/ultra/scenarios/common/distributions.py:149]: The 200 social vehicles is still large amount, however I sort of accounted for some worst case scenarios. It is a bit hard to explain here, but I can reduce the number to 100 even?

Comment (christianjans) [reply ultra/ultra/scenarios/common/distributions.py:149]: Oh okay, I see. Even with the changes of #1014, the scenario will not run out of social vehicles?

Comment (JenishPatel99) [reply ultra/ultra/scenarios/common/distributions.py:149]: Even with `endless_traffic` turned off, we have a sufficient amount of vehicles to entertain for more 200 secs (2000 timesteps)

**Review by christianjans — APPROVED** [review]

**Pull request merged**